### PR TITLE
Fix firefox - fails silently on bad mime type

### PR DESCRIPTION
**Pull request by soyuka**
Now FF is working. Though as you said on twitter I experienced bad behavior using FF and an http-based signaling server.
I can propose a patch to use https://github.com/soyuka/signalhubws. lmk

### CODE REVIEW

**Comment by louiscenter** [comment]
@soyuka thank you for submitting a PR! should this `mime` variable you've created be used somewhere in `broadcast.js`?

A PR to switch to `signalhubws` would also be very cool :)

**Comment by soyuka** [comment]
Nice catch, fixed the watch file also!

PRs comin

File: lib/getMimeType.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+var MIME_TYPES = ['video/webm;codecs=vp9,opus', 'video/webm;codecs=vp8,opus']
```
Comment (soyuka) [inline lib/getMimeType.js:1]: maybe you prefer lib/utils.js to put this?

Comment (louiscenter) [reply lib/getMimeType.js:1]: @soyuka it's probably fine for the moment. I can modularize things later on if files become too hectic.

Comment (louiscenter) [reply lib/getMimeType.js:1]: @soyuka ah, I see you've already done it. even better! thank you ^__^

**Pull request merged**